lib/rules/no-empty-component-block.js

```diff
@@ -26,6 +27,17 @@ function hasAttributeSrc(componentBlock) {
   return hasAttribute && hasSrc
 }
 
+/**
+ * check whether value under the component block is only whitespaces or break lines
+ */
+function isValueOnlyWhiteSpacesOrLineBreaks(componentBlock) {
+  return (
+    componentBlock.children.length === 1 &&
+    componentBlock.children[0].type === 'VText' &&
+    /^(\s|\n)+$/.test(componentBlock.children[0].value)
```

Comment:
I'm not sure if this regex is cover all cases...

Reply:
Maybe using `/^\s*$/` will also work.
However, I'm not good at using regular expressions, so I often do the following.

```js
!componentBlock.children[0].value.trim()
```

Reply:
It seems nice.
I reflected this code.